{"run_number": "1660329951175058", "step_name": "start", "task_id": "1", "attempt_id": 0, "name": "_exception", "content_type": "gzip+pickle-v2", "type": "metaflow.artifact", "sha": "f3627f46179fdd95bf0e83101840fd1d71b60e40", "ds_type": "local", "location": ":root:/home/skyolia/JupyterProjects/time_series/mmm/.metaflow", "flow_id": "TrainTestFlow", "user_name": "skyolia", "tags": [], "system_tags": ["python_version:3.8.5", "user:skyolia", "metaflow_version:2.7.2", "runtime:dev"], "ts_epoch": 1660329952100}